content/filters/additional_filters.md

```diff
@@ -0,0 +1,155 @@
+# Additional Filters
+
+## url_encode
+
+Check does source can be encode for URI link.
+
+#### Input
+
+~~~ liquid
+  <!-- Rental Headline = "Super Headline" -->
+    {{ rental.headline | url_encode }}
+~~~
+
+#### Output
+
+~~~
+  Super%20Headline
+~~~
+
+## rental_photo_url
+
+Load photo with given size.
+
+#### Input
+
+~~~ html
+  <img src="{{ photo | rental_photo_url: 'large' }} 1200w" >
+~~~
+
+#### Output
+
+~~~ html
+  <img src=".../image/upload/c_fill,fl_progressive,g_center,h_800,q_90,w_1200/v1436964916/uw5bsnqqbkwtxfqocbjd.jpg 1200w" >
+~~~
+
+## category_photo_url
+
+Similiar to `rental_photo_url`. In Category, upolad photo with specified size.
+
+## convert_stars
+
+Multiply rating by 2. Round and multiply by 10.
```

Comment:
Done. I've used your sentence: `Converts rental rating to be used as star rating.`